packages/native-stack/src/views/NativeStackView.tsx

```diff
@@ -86,6 +93,10 @@ export function NativeStackView({ state, descriptors }: Props) {
 
         const nextPresentation = nextDescriptor?.options.presentation;
 
+        const shouldBePreloaded =
```

Comment:
I think `isPreloaded` is better. The screen is already preloaded in the router, the view layer needs to handle it.

```suggestion
        const isPreloaded =
```